src/fasd_bot/main.rb

```diff
@@ -18,7 +18,7 @@
 
 Telegram::Bot::Client.run(Config::TELEGRAM_TOKEN, timeout: 60) do |bot|
   bot.listen do |message|
-    next unless message.text
+    next unless message&.text
```

Comment:
hm, good point. Can't think of what is causing this. Looks like it was happening even before my changes 🤔 

But since this is making the pipeline fail with exit code (1), I'll get this merged and see if the issue gets resolved.